smart-contracts/contracts/BridgeBank/BridgeBank.sol

```
@@ -76,6 +130,30 @@ contract BridgeBank is BankStorage,
         _;
     }
 
+    /*
+     * @dev: Set the token address in Eth whitelist
+     *
+     * @param _token: ERC 20's address
+     * @param _inList: set the _token in list or not
+     * @return: new value of if _token in whitelist
+     */
+    function updateEthWhiteList(address _token, bool _inList)
+        public
+        onlyOperator
+        returns (bool)
+    {
+        // Do not allow a token with the same address to be whitelisted
+        if (_inList) {
+            // if we want to add it to the whitelist, make sure it's not there yet
+            require(!getTokenInEthWhiteList(_token), "whitelisted");
+            require(!getCosmosTokenInWhiteList(_token), "whitelisted");
```

This error message should throw an error like so:
`already in cosmos whitelist`
so that it denotes the whitelist it is a part of.